plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergPageSinkProvider.java

```diff
@@ -91,6 +91,7 @@ private ConnectorPageSink createPageSink(ConnectorSession session, IcebergWritab
                 jsonCodec,
                 session,
                 tableHandle.getFileFormat(),
+                tableHandle.getStorageProperties(),
```

Comment:
how is a user expected to set storage properties?

Reply:
@findepi The whole table property map is passed to the writable file handle in `IcebergMetadata`. I am passing it to the `PageSink` constructor to derive the metrics config property.

Reply:
sure, but that doesn't answer my question -- how a Trino user can set this? how an Iceberg user would do that?

Reply:
Users can set it in Spark.

Reply:
_how_ do they do that?

then, if this is useful, _how_ users will do this via Trino?

Reply:
Using these table properties: https://github.com/apache/iceberg/blob/master/core/src/main/java/org/apache/iceberg/TableProperties.java#L187-L189

And the compute engine or user calls the Iceberg API like:

```java
table.updateProperties()
.set("write.metadata.metrics.default", "full")
.commit()
```

I think just respecting the metrics configuration on Trino side is a good start in case the same config is set in Spark. I don't know if you would want users to set this on Trino side, because it seems like you want the file writing process to be more managed instead of users changing the write configs here and there. But if Trino is open for configuring this part, then we can use the same mechanism and introduce corresponding Iceberg table properties, maybe through 1 property for table default, 1 map property for column defaults.